pagopa/io-web-profile
# [IOPID-698] : Removing Console JS Warning

## Short description

The purpose of these modifications is to remove the security warning generated by footer component in the browser console

## List of changes proposed in this pull request

- Updated dependency versions
- Added an empty href to the missing button, which was causing the warning

## Note

I've added the comment **'FIXME IOPID-566**' to the code (footer.tsx) in order to address this later when we'll integrate analytics. At that moment, we'll could take advantage to understand how to integrate with OneTrust (if needed) and decide which value should be passed to the href or if we need to utilize any specific action.

## How to test

Run the project in dev mode, open the browser, go to main page and check if we still have security js warning in console.
